pkg/common/operator_serviceaccount.go

```diff
@@ -0,0 +1,36 @@
+package common
```

Comment:
Please add the copyright and License to this file, could you also add it to operator_namespace.go since we missed that also.

Reply:
Copyright headers are added.